docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+services:
+  node: 
+    image: victorabz/the-spinners
+    build: .
+    ports: 
+      - 80:3000
```

Comment:
Port 80 is usually not available on the host machine. You can change it to `3000`